pycaption/base.py

```diff
@@ -4,7 +4,8 @@
 
 from .exceptions import CaptionReadError, CaptionReadTimingError
 
-DEFAULT_LANGUAGE_CODE = 'en-US'
+# `und` a special identifier for an undetermined language according to ISO 639-2
+DEFAULT_LANGUAGE_CODE = 'und'
```

Comment:
this might break backwards compatibility, so we should also make it a configurable option for those who would like to preserve the old default.